lib/persisted-model.js

```diff
@@ -1527,6 +1546,107 @@ module.exports = function(registry) {
     });
   };
 
+  /**
+   * Subscribe to changes of this model.
+   *
+   * @param {Object} options
+   * @param {Object} options.where Only changes to models matching this where filter will be included in the `ChangeStream`.
+   * @callback {Function} callback
+   * @param {Error} err
+   * @param {ChangeStream} changes
+   */
+
+  PersistedModel.subscribe = function(options, cb) {
+    if(typeof options === 'function') {
+      cb = options;
+      options = undefined;
+    }
+
+    var idName = this.getIdName();
+    var Model = this;
+    var changes = new PassThrough({objectMode: true});
+    var writeable = true;
+    var ref = {changes: changes};
+
+    changes.destroy = function() {
+      changes.removeAllListeners('error');
+      changes.removeAllListeners('end');
```

Comment:
It would be good to remove hook observers too.

I think that otherwise the list of registered hook observers will grow without limits, each DAO operation will have to invoke a lot of inactive hooks and the server will gradually slow down.